integration-tests/actions/vrf/vrfv2/contract_steps.go

```diff
@@ -727,7 +387,26 @@ func FundSubscriptions(
 	return nil
 }
 
+func FundVRFCoordinatorV2Subscription(
+	linkToken contracts.LinkToken,
+	coordinator contracts.VRFCoordinatorV2,
+	chainClient blockchain.EVMClient,
+	subscriptionID uint64,
+	linkFundingAmountJuels *big.Int,
+) error {
+	encodedSubId, err := chainlinkutils.ABIEncode(`[{"type":"uint64"}]`, subscriptionID)
+	if err != nil {
+		return fmt.Errorf("%s, err %w", vrfcommon.ErrABIEncodingFunding, err)
+	}
+	_, err = linkToken.TransferAndCall(coordinator.Address(), linkFundingAmountJuels, encodedSubId)
+	if err != nil {
+		return fmt.Errorf("%s, err %w", vrfcommon.ErrSendingLinkToken, err)
+	}
+	return chainClient.WaitForEvents()
```

Comment:
So we are just waiting for any kind of event? We are not subscribing to specific ones related to transferring LINK tokens to our subscription? If the contract would emit a different event, would this unblock the `WaitForEvents` call as well?

Reply:
I dont check for the event as I think it is not needed, as it would fail with an error and Im checking the error - line 402